src/all/onepace/src/eu/kanade/tachiyomi/animeextension/all/onepace/Onepace.kt

```diff
@@ -26,7 +26,7 @@ import org.jsoup.nodes.Element
 import uy.kohesive.injekt.Injekt
 import uy.kohesive.injekt.api.get
 import uy.kohesive.injekt.injectLazy
-import java.lang.Exception
+import kotlin.Exception
```

Comment:
Fun fact you don't have to import this, it's an inbuilt type
But it doesn't really matter